streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/system/service/VariableService.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.streampark.console.system.service;
+
+import org.apache.streampark.console.base.domain.RestRequest;
+import org.apache.streampark.console.core.entity.Application;
+import org.apache.streampark.console.system.entity.Variable;
+
+import com.baomidou.mybatisplus.core.metadata.IPage;
+import com.baomidou.mybatisplus.extension.service.IService;
+
+import java.util.List;
+
+public interface VariableService extends IService<Variable> {
+
+    /**
+     * find variable
+     *
+     * @param variable        variable
+     * @param restRequest queryRequest
+     * @return IPage
+     */
+    IPage<Variable> findVariables(Variable variable, RestRequest restRequest);
+
+    /**
+     * get variables through team
+     * @param teamId
+     * @return
+     */
+    List<Variable> findByTeamId(Long teamId);
+
+    /**
+     * create variable
+     *
+     * @param variable variable
+     */
+    void createVariable(Variable variable) throws Exception;
+
+    /**
+     * delete variable
+     *
+     * @param variable variable
+     */
+    void deleteVariable(Variable variable) throws Exception;
+
+    /**
+     * update variable
+     *
+     * @param variable variable
+     */
+    void updateVariable(Variable variable) throws Exception;
+
+    Variable findByVariableCode(Long teamId, String variableCode);
```

Comment:
findByTeamIdAndVariableCode?

Reply:
I have really thought about this naming, and also referred to the writing method of team. Considering that team is public in variable management, there is no name to add teamId to it. Is it OK to keep it unchanged?

MemberService：
Member findByUserName(Long teamId, String userName);

Reply:
Actually, the method is `findByTeamAndUserName` when I develop the team management. But @wolfboys change it from `findByTeamAndUserName` to `findByUserName` in #1792 . I don't know why. 

Hi @wolfboys , could you help explain why? Is it a mistake, or is there another reason?

![image](https://user-images.githubusercontent.com/38427477/195992104-aebdaf9f-c675-4029-a0c1-70ee99fac58d.png)

Reply:
> Actually, the method is `findByTeamAndUserName` when I develop the team management. But @wolfboys change it from `findByTeamAndUserName` to `findByUserName` in #1792 . I don't know why.
> 
> Hi @wolfboys , could you help explain why? Is it a mistake, or is there another reason?
> 
> ![image](https://user-images.githubusercontent.com/38427477/195992104-aebdaf9f-c675-4029-a0c1-70ee99fac58d.png)

In fact, this method is used for business. The use of teamId has been clarified. I don’t think it is necessary to have a method name that is too long. For example, if you query users based on roleId, teamid, and userName, should the method name be written like this? 
`findByRoleAndTeamAndUserName(...)`

if there are more parameters, how do you define the method name?

Reply:
Thanks for your clarification. LGTM